CDCgov/prime-reportstream
# 5306 Removed meta from settings API response

This PR removes a redundant object in the API response called meta.  The data from the old meta object is now in the root of the object with the rest of the response.

Test Steps:
1. Build this branch and run your local ReportStream.
2. Access the following endpoints and verify that there is no meta object that is part of the response (or each object in the response).
```
http://localhost:7071/api/settings/organizations
http://localhost:7071/api/settings/organizations/ignore
http://localhost:7071/api/settings/organizations/ignore/senders
http://localhost:7071/api/settings/organizations/ignore/senders/default
http://localhost:7071/api/settings/organizations/ignore/receivers
http://localhost:7071/api/settings/organizations/ignore/receivers/HL7
```

## Changes
- Removed meta object from settings API response

## Checklist

### Testing

### Process

## Linked Issues
- #5306

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
